src/aap_eda/services/activation/db_log_handler.py

```diff
@@ -47,6 +47,7 @@ def write(
         lines: Union[list[str], str],
         flush: bool = False,
         timestamp: bool = True,
+        log_timestamp: int = 0,
```

Comment:
This field is not used and doesn't look like we need to keep the state of this, don't?

Reply:
The Podman and Kubernetes engines need it when inserting log lines into DB.

Reply:
where is used? 
I don't see any reference to LogHandler.log_timestamp, always is passed through function arguments which makes sense to me.

Reply:
nevermind, the GH view has confused to me and I thought it is in the __init__.